### ISSUE TITLE
The issues encountered when running the command.

### ISSUE THREAD

**Issue by zsjiang99** [baichen318/boom-explorer-public#3]
I would like to ask if you have encountered this issue when running this command, and if you have any good suggestions?
![image](https://github.com/baichen318/boom-explorer-public/assets/97930258/5cda7f32-9ad6-4089-af69-a8aabb6d7cef)

**Comment by baichen318** [comment]
Hi @luck-codeer ,

I have not encountered this issue before. Regarding the message, it could be a misaligned `botorch` version.

In my environment, my `botorch` is version: `0.6.0`.

I will update my executable environment to this repository, shortly.

Best regards,
Chen

**Comment by zsjiang99** [comment]
Thank you very much, I have solved this problem, thank you.

**Comment by Lmywl** [comment]
I encounter the same question too，can you tell me how you deal with that .
appreciate it !




> I would like to ask if you have encountered this issue when running this command, and if you have any good suggestions? ![image](https://user-images.githubusercontent.com/97930258/260880340-5cda7f32-9ad6-4089-af69-a8aabb6d7cef.png)

**Comment by zsjiang99** [comment]
Botorch version incompatibility issue may arise, as some functions from the old versions of Botorch may not be present in the new ones. It is recommended to use the version of Botorch that matches the one used by the author at the time of writing. I remember I installed Botorch 0.6.0, Gpytorch 1.6.0, and Torch 1.9.0. You can try using these versions instead.

**Comment by Lmywl** [comment]
> Botorch version incompatibility issue may arise, as some functions from the old versions of Botorch may not be present in the new ones. It is recommended to use the version of Botorch that matches the one used by the author at the time of writing. I remember I installed Botorch 0.6.0, Gpytorch 1.6.0, and Torch 1.9.0. You can try using these versions instead.

appreciate it , I'll try it

**Comment by lpj20** [comment]
Hi, I also encountered this problem. And @luck-codeer gave the correct solutions. However, when I installed the botorch by source code directly, there was a problem with the version of setuptools and it was difficult to find the reason.
Below is the correct installation method:
pip install botorch==0.6.0 (it will install gpytorch with 1.11.0 by default)
pip uninstall gpytorch
pip install gpytorch==1.6.0
